src/main/java/com/mykaarma/urlshortener/util/UrlServiceUtil.java

```diff
@@ -158,7 +154,7 @@ public boolean isHashValid(String hash) throws ShortUrlException {
 		if(hash.length()>20||hash.isEmpty()) {return false;}
 		for(int i=0;i<hash.length();i++)
 		{
-			if(!((hash.charAt(i)>='a'&&hash.charAt(i)<='z')||(hash.charAt(i)>='A'&&hash.charAt(i)<='Z')||(hash.charAt(i)>='0'&&hash.charAt(i)<='9')||(hash.charAt(i)=='+')||(hash.charAt(i)=='-')))
+			if(!((hash.charAt(i)>='a'&&hash.charAt(i)<='z')||(hash.charAt(i)>='A'&&hash.charAt(i)<='Z')||(hash.charAt(i)>='0'&&hash.charAt(i)<='9')))
```

Comment:
We can perhaps simplify this characters check by using regex. Besides, what is the point of having `randomAlphabet` as user input, if we just what to limit the characters using this check?